Microsoft.HealthVault.Fhir/Transformers/ApproximateDateTimeToFhir.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using Hl7.Fhir.Model;
+using Microsoft.HealthVault.ItemTypes;
+
+namespace Microsoft.HealthVault.Fhir.Transformers
+{
+    public static partial class ItemBaseToFhir
+    {
+        // Register the type on the generic ItemBaseToFhir partial class
+        public static FhirDateTime ToFhir(this ApproximateDateTime approximateDateTime)
+        {
+            return ApproximateDateTimeToFhir.ToFhirInternal(approximateDateTime);
+        }
+    }
+    internal static class ApproximateDateTimeToFhir
+    {
+        internal static FhirDateTime ToFhirInternal(ApproximateDateTime approximateDateTime)
+        {
+            if (!approximateDateTime.ApproximateDate.Month.HasValue && !approximateDateTime.ApproximateDate.Day.HasValue)
```

Comment:
Add open and closing curly braces everywhere on this file. Even when we have only one statement the guidance is to open and close braces for readibility.